processor/servicegraphprocessor/testdata/service-graph-config.yaml

```diff
@@ -0,0 +1,23 @@
+receivers:
+  nop:
+
+exporters:
+  nop:
+
+processors:
+  servicegraph:
+    metrics_exporter: metrics
```

Comment:
Should there be a metrics exporter here in the config?

Reply:
In a valid collector config, yes. In this case it's only validating that the processor's config is correct.

Reply:
Not really. This yaml is for a test checking that the processor's config is correct, not the entire pipeline.